windows/win_unzip.ps1

```diff
@@ -57,17 +57,36 @@ $recurse = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "recurse" -defaul
 $rm = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "rm" -default "false")
 
 If ($ext -eq ".zip" -And $recurse -eq $false) {
-    Try {
-        $shell = New-Object -ComObject Shell.Application
-        $zipPkg = $shell.NameSpace([IO.Path]::GetFullPath($src))
-        $destPath = $shell.NameSpace([IO.Path]::GetFullPath($dest))
-        # 20 means do not display any dialog (4) and overwrite any file (16)
-        $destPath.CopyHere($zipPkg.Items(), 20)
-        $result.changed = $true
+    If (-Not (Get-WindowsFeature -Name 'Server-Gui-Shell')) {
```

Comment:
We may want to resort to just checking if it's installed,  If so then use the Assembly.

`win_feature` could be used to download the feature, and if it exists then we have the check in this module to load the Assembly and use it.  

In other words, I don't think we should resort to auto-installing the feature in this module.  It isn't this module's job to do those sorts of things and I think most people are against modules so (unless it is explicitly their job i.e `win_msi`, `win_feature`, `win_chocolatey` etc..).

Thoughts?